Sources/BazelizeKit/Kit.swift

```diff
@@ -33,17 +33,26 @@ public final class Kit {
         let targets = project.targets.compactMap {
             $0 as? XCode.Target
         }
-        for target in targets {
-            try target.generateBUILD(self)
-        }
 
-        let spm_repositories = targets.spm_repositories(project.projectPath)
+        /// {WORKSPACE}/WORKSPACE
+//        let spm_repositories = targets.spm_repositories(project.projectPath)
         let workspace = Workspace { builder in
             builder.default()
-            builder.custom(code: spm_repositories)
+//            builder.custom(code: spm_repositories)
         }
         try? workspace.generate(project.workspacePath)
 
+        /// {WORKSPACE}/BUILD
+        try? project.generateBUILD(self)
+
+        /// {WORKSPACE}/.bazelrc
+        try? project.generateBazelRC(self)
```

Comment:
Is `generateBazelRC` a must for this tool?

I'm a bit worried about whether it would have conflicts with other customization by the future consumers.

Reply:
Generate `.bazelrc` can be optional. 

Our main target is `WORKSPACE` & `BUILD`